### ISSUE TITLE
Photos inside an hidden folder appears inside the Photos app

### ISSUE THREAD

**Issue by MrAlucardDante** [nextcloud/photos#1061]
**Describe the bug**
Photos inside an hidden folder appears inside the Photos app.
The issue only seems to be on the web version (nextcloud server). 
The media tab in the nextcloud app doesn't have this issue, but I don't know if it uses the photos app (probably not since the ios app has its own repo)

**To Reproduce**
1. Create an hidden folder (name starts .)
2. Add photos inside that folder
3. Go to the Photos app
4. Go to your folders
5. Navigate to the hidden folder

**Expected behavior**
Hidden folders and files inside those folders shouldn't appear. The Files app as a setting to show/hide hidden folders

**Screenshots**
![image](https://user-images.githubusercontent.com/20476382/159157618-ee9ccde8-aae7-451d-ab2f-9802f457a517.png)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser : Firefox
 - Version : Nextcloud v23.0.2, Photos app v1.5.0

**Comment by Tealk** [comment]
Yes I would also like to have the function that images from a hidden folder are not displayed.
You don't hide folders for no reason.

**Comment by Jerome-Herbinet** [comment]
I absolutely have the same needs and suggestion. 

In my case, I have hidden folders in almost each photo folder:
- .trash (duplicate photos or photos with flaws that I want to keep out of sight without actually deleting them)
- .darktable_exported (optimized versions of photos processed and exported from Darktable, which are usually shared and that I don’t want to see by default in Nextcloud Photos) 

Being able to hide these hidden folders and their contents would let me avoid seeing a whole set of photos I don’t want to see, starting with the duplicates (original version vs. photo processed in Darktable)

I suggest adding an option to specify whether or not to display hidden folders.
